tests/dielectric_profile/one_layer.cpp

```diff
@@ -126,6 +150,24 @@ double erf_deriv(double point, double e1, double e2, double w, double c) {
   return (factor * val);
 }
 
+double log_value(double point, double e1, double e2, double w, double c) {
+  w /= 6.0;
+  double epsLog = std::log(e2 / e1);
+  double val = (1.0 + boost::math::erf((point - c) / w)) / 2.0;
```

Comment:
Use `pcm::erf` instead of `boost::math::erf` directlty. It will make easier pulling out Boost later on.